stories/uri_components/Visualization.tsx

```diff
@@ -407,6 +408,7 @@ storiesOf("URI components", module)
                     identifier="pushpin-chart-location-size-color-segment-tooltip"
                     config={{
                         mapboxToken: process.env.STORYBOOK_MAPBOX_ACCESS_TOKEN,
+                        tooltipText: attribute("/gdc/md/storybook/obj/24.df").localIdentifier("tooltipText"),
```

Comment:
this conversation was done [here](https://github.com/gooddata/gooddata-react-components/pull/1573/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-96359152e9393aae1269513293326debR119), right?

So we only pass the uri to tooltipText is ok
`tooltipText: "/gdc/md/storybook/obj/24.df"`